.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,18 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+if [ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
```

Comment:
I think it should be:

```
if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
```
(one leading bracket is missing)


Also can you add info log?
```
echo "Injecting snippet mapping data"
```

Reply:
Done.